joshuaCarpentier/assignments/fastSlow/lc141/lc141.js

```diff
@@ -14,18 +48,30 @@ class Node {
 }
 
 const has_cycle = function(head) {
-  // TODO: Write your code here
+  let slow = head,
+      fast = head;
 
+      while (fast !== null && fast.next !== null){
+        fast = fast.next.next;
+        slow = slow.next;
+        if (slow === fast){
+          return true;
+        }
+      }
+
+// dont necessarly need to have else when just returning boolean value
   return false
 }
```

Comment:
Code looks good! Small feedback would be not to leave comments in your codebase unless absolutely necessary & where is the love for typing it twice?!? #LEGGO #GoodWork #PracticeHowYouPlay #TypeTwice